slugbuilder/artifact/main.go

```diff
@@ -12,6 +12,10 @@ import (
 	"net/http"
 	"os"
 	"os/exec"
+	"path/filepath"
+	"strings"
+
+	"gopkg.in/yaml.v2"
 
```

Comment:
Hmm `goimports` seems to want to put a new line in there on my machine. Life is too short to figure out why 😛